presto-docs/src/main/sphinx/release/release-0.275.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.275
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator to correctly identify running queries as leaked.
+* Improve the memory limit error message with allocationTag.
```

Comment:
"Enrich error message when exceeding memory limit by adding more information on ...". `allocationTag` is too internal a thing that users probably don't understand.